core/src/main/java/org/apache/accumulo/core/clientImpl/bulk/BulkImport.java

```diff
@@ -482,12 +484,16 @@ private SortedMap<KeyExtent,Bulk.Files> computeMappingFromFiles(FileSystem fs, T
     if (this.executor != null) {
       executor = this.executor;
     } else if (numThreads > 0) {
-      executor = service = context.threadPools().getPoolBuilder("BulkImportThread")
-          .numCoreThreads(numThreads).build();
+      executor = service =
+          context.threadPools().getPoolBuilder(METRICS_POOL_PREFIX + "bulk.import.service")
```

Comment:
This is a client side thread pool for bulk load, could use the following name

```suggestion
          context.threadPools().getPoolBuilder(METRICS_POOL_PREFIX + "client.bulk.load")
```

Reply:
Fixed in 21aa29c435